### ISSUE TITLE
Issue when transfering propriety from a user to an organisation?

### ISSUE THREAD

**Issue by M5oul** [gogits/gogs#2478]
When transfering propriety of a repository from a Gogs user to a Gogs organisation, the address of the repository of the mirror now looks: `https://user:@github.com/<org>/<repo>`

`user:@` part isn't present on a new mirror. with `user`where gogs user,
Is there an issue?

I am using Gogs 0.8.10.

**Comment by bkcsoft** [comment]
I can't reproduce this. The repo, is it a mirror or a plain repo?

**Comment by bkcsoft** [comment]
Can't reproduce with a mirrored repo either...

**Comment by Unknwon** [comment]
Thanks your feedback!

> the address of the repository of the mirror now looks: https://user:@github.com/<org>/<repo>

Can you rephrase this sentence?

**Comment by M5oul** [comment]
address of the repository mirrored is: https://user:@github.com//.

user:@ shouldn't be here.

I couldn't reproduced it.

**Comment by Unknwon** [comment]
@M5oul Can you try to test on `master` branch version see if this problem still exists?

**Comment by M5oul** [comment]
I can't reproduce it. I close the issue.
